security-analysis/security-analysis-api/src/test/resources/SecurityAnalysisParameters.json

```diff
@@ -1,5 +1,10 @@
 {
-  "version" : "1.0",
+  "version" : "1.1",
+  "increasedFlowViolationsThreshold" : "0.1",
+  "increasedLowVoltageViolationsThreshold" : "0.0",
```

Comment:
Better to use other values than 0 for all params, to ensure it's not the default values